parse/helpers.js

```diff
@@ -241,10 +241,19 @@ export let getComment = (line) => {
 export let getData = (maybeProp) => {
   if (!maybeProp) return null
 
-  let match = maybeProp.value.match(/^((show|capture)\s+)?(.+)$/)
+  let match = `${maybeProp.value}`.match(/^((show|capture)\s+)?(.+)$/)
   if (!match) return null
 
   let path = match[3]
+  if (/^\"(.+)?\"$/.test(path) || /^\'(.+)?\'$/.test(path)) {
+    return {
+      value: path,
+      isConstant: true,
+    }
+  }
+  if (!isNaN(Number(path))) {
```

Comment:
Should we force all the values to be wrapped in quotes?

Reply:
Not sure I fully get your point here. How do we distinguish then if a value is supposed to be a number or a string? e.g. `10` vs `"10"`? Just thinking if in some circumstances we'll need to have a string even though the content consists on numbers only (e.g. maybe an ID could be an example)

Reply:
Ah that's a fair point... I was coming from a different angle, in which all the values passed as constants are the same but you're right, we'll be missing this case otherwise. Let's keep it as is then. Good catch!